libs/api/prisma/migrations/20230126155832_session_on_delete_cascade/migration.sql

```diff
@@ -0,0 +1,56 @@
+-- DropForeignKey
+ALTER TABLE "articles.revisions.author" DROP CONSTRAINT "articles.revisions.author_authorId_fkey";
```

Comment:
Theres a lot going on here that shouldnt be here. 

Have you had by chance the GUID branch checked out once and ran migrations there?